tests/js/spec/utils/replays/getPrevUserAction.spec.tsx

```diff
@@ -0,0 +1,126 @@
+      type: BreadcrumbType.UI,
+    },
+    {
+      category: 'ui.input',
+      color: 'purple300',
+      data: undefined,
+      description: 'User Action',
+      event_id: null,
+      id: 4,
+      level: BreadcrumbLevelType.INFO,
+      message: 'div.App > section.padding-b-2 > div.makeStyles-search-input-2 > input',
+      timestamp: '2022-05-04T19:47:11.086000Z',
+      type: BreadcrumbType.UI,
+    },
+    {
+      category: 'ui.click',
+      color: 'purple300',
+      data: undefined,
+      description: 'User Action',
+      event_id: null,
+      id: 20,
+      level: BreadcrumbLevelType.INFO,
+      message: 'div.App > section.padding-b-2 > div.makeStyles-search-input-2 > input',
+      timestamp: '2022-05-04T19:47:52.915000Z',
+      type: BreadcrumbType.UI,
+    },
+    {
+      category: 'navigation',
+      color: 'green300',
+      data: {
+        from: '/organizations/sentry/user-feedback/?project=6380506',
+        to: '/organizations/sentry/issues/',
+      },
+      description: 'Navigation',
+      event_id: null,
+      id: 166,
+      level: BreadcrumbLevelType.INFO,
+      message: undefined,
+      timestamp: '2022-05-04T19:47:59.915000Z',
+      type: BreadcrumbType.NAVIGATION,
+    },
+  ];
+}
+
+describe('getPrevUserAction', function () {
+  it(`should return the previous user action even if the timestamp
+    is closer to the next action`, function () {
+    const userActionCrumbs: Crumb[] = createUserActionCrumbs();
+    const results = getPrevUserAction(
+      userActionCrumbs,
+      START_TIMESTAMP_MS,
+      CURRENT_TIME_MS
+    );
+
+    expect(results?.id).toEqual(4);
+  });
+
+  it('should return undefined when userActions is not defined', function () {
+    const userActionCrumbs = [];
+    const results = getPrevUserAction(
+      userActionCrumbs,
+      START_TIMESTAMP_MS,
+      CURRENT_TIME_MS
+    );
+
+    expect(results).toBeUndefined();
+  });
+
+  it('should return undefined when startTimestamp is not defined or is equal to 0', function () {
+    const userActionCrumbs: Crumb[] = createUserActionCrumbs();
+    const results = getPrevUserAction(userActionCrumbs, 0, CURRENT_TIME_MS);
+
+    expect(results).toBeUndefined();
+  });
+
+  it('should return undefined when userActions has only item and the current time is before that item', function () {
+    const userActionscrumbs: Crumb[] = createUserActionCrumbs().slice(4, 5);
+    const results = getPrevUserAction(
+      userActionscrumbs,
+      START_TIMESTAMP_MS,
+      CURRENT_TIME_MS
+    );
+
+    expect(results).toBeUndefined();
+  });
+
+  it('should return the user action when timestamp matches the timestamp of a breadcrumb', function () {
+    const userActionCrumbs: Crumb[] = createUserActionCrumbs();
+    const exactCrumbTime: number = 8135;
+    const results = getPrevUserAction(
+      userActionCrumbs,
+      START_TIMESTAMP_MS,
+      exactCrumbTime
+    );
+
+    expect(results?.id).toEqual(4);
+  });
```

Comment:
This asserts that if you hover over a timestamp we should pick the crumb at that timestamp.

I think that's not the right behavior.
Consider we have a 3minute replay, and three events: `A @ 0:00`, `B @ 1:00` and `C @ 2:00`. The replay is paused.
1. If we hover over the end of the replay, at time=3:00 then `C` will be highlighted.
2. If we click to jump to event `C` then the current time will be set to `2:00`
3. If we hover again at the `2:00` mark then event `C` is still highlighted. 
This is a problem because we're not selecting the event _before_ `2:00`, we're actually selecting the at `2:00`.